src/main/webapp/css/user-page.css

```diff
@@ -47,8 +48,22 @@
   max-width: 75%;
 }
 
+.centered{
+  text-align: center;
+}
+
+.padded {
+  padding: 10px;
+}
+
+.rounded {
+  border: thin solid black;
+  border-radius: 5px;
+  overflow: hidden;
+}
+
 /* Use this to hide certain elements like forms if the
    user is not logged in or viewing their own page. */
 .hidden {
   display: none;
-}
```

Comment:
No newline at end of file. I don't think this is a big deal... I believe it's just a message to make it clearer that there isn't one when trying to distinguish between different files.

Reply:
Thanks for spotting that, Elaine. You're right that this isn't a big deal - Sabin, you don't need to worry about this.